src/components/visible.js

```diff
@@ -2,21 +2,17 @@ var registerComponent = require('../core/component').registerComponent;
 
 /**
  * Visibility component.
- *
- * @param {bool}
  */
 module.exports.Component = registerComponent('visible', {
-  schema: { default: true },
+  flat: true,
 
-  update: function () {
-    this.el.object3D.visible = this.data;
-  },
+  type: 'boolean',
 
-  parse: function (value) {
-    return value !== 'false';
+  schema: {
```

Comment:
Is a ```flat``` flag necessary? We could infer from the schema if there's only one attribute named ```default```